app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -171,10 +171,10 @@ def self.key_pair_type
   def parse_image(image, is_public)
     uid      = image.image_id
     location = image.image_location
-    guest_os = (image.platform == "windows") ? "windows" : "linux"
-    if guest_os == "linux"
+    guest_os = (image.platform == "windows") ? "windows_generic" : "linux_generic"
```

Comment:
@blomquisg any idea why we used linux&windows as guest os? The allowed normalized names are only windows_generic and linux_generic, from https://github.com/Ladas/manageiq/blob/6eeec01542e27442e0d6515c39417abc0470b42b/app/models/operating_system.rb#L13